### PR TITLE
Spaces Removal

### PR DESCRIPTION
**Pull request by klorpa**
#### Summary

```SUMMARY: Bugfixes "Removes several unneeded spaces."```

### CODE REVIEW

File: data/mods/Magiclysm/lore.md [hunk 1]
```diff
@@ -31,5 +31,4 @@ In the case of "how is a spell part of a class?" the spell's characteristics det
 - Biomancer: Flesh mage. Manipulating the caster's flesh is easier but higher tier spells can manipulate others' flesh. Features Acid.
 - Druid: Plant mage. Communes with nature. The typical "druid" of old is usually a combination of this class and Earthshaper. Has a slight secondary affinity with water.
 
-
```
Comment (KorGgenT) [inline data/mods/Magiclysm/lore.md:34]: don't you need the double space in markdown to get a line break?

**Comment by esotericist** [comment]
Generally corrections of this sort which do not actually impact content are reserved for when there are actual content changes, so that `git blame` and the such return correct information.

Conversely, this PR produces an absurd amount of noise for tracking actual content history for no obvious reason.

Is this actually in the service of anything?

**Comment by jbytheway** [comment]
I'm in favour of cleaning up our style, but only if this style becomes enforced by the CI.  For example, I'd be happy to see a PR which added flake8 verification of the Python scripts to the CI, together with associated cleanup.  Cleanup without enforcement is not going to stick.

**Comment by kevingranade** [comment]
Agree with @jbytheway, this isn't going to meaningfully fix anything and 'd rather not churn code over it if we can't make it stick.

**Pull request closed without merging**